app/create_file.py

```diff
@@ -1 +1,48 @@
-# write your code here
+import sys
+import os
+import datetime
+
+
+def create_file(file_name: str) -> None:
+    with open(file_name, "a") as file:
+        file.write(
+            str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S\n")))
+        line_number = 1
+        while True:
+            input_content = input("Enter content line:")
```

Comment:
Add a space after `:`
```suggestion
            input_content = input("Enter content line: ")
```